pages/docs/balance/open-banking/plaid.mdx

```diff
@@ -40,25 +40,55 @@ Use the following credentials to simulate successful authentication with test ba
 
 Plaid offers two test banks: one for **checking** accounts and one for **savings** accounts. Both have predefined account and routing numbers. This means:
 
 - You can add up to two banks per customer (one checking, one savings).
 - To test additional banks, remove an existing funding source and repeat the Plaid Link flow.
 
+### Plaid SDKs for Account Authentication
+
+Plaid Link serves as the front-end widget for Plaid IAV, enabling users to securely authenticate and link their bank accounts. This widget can be seamlessly embedded into your application using one of [Plaid’s Link client SDKs](https://plaid.com/docs/api/libraries/#link-client-sdks).
+
+On this guide, we’ll be using the `react-plaid-link` package, which is specifically designed for React web applications. It simplifies the integration of Plaid Link by providing a React hook to handle initialization and user interactions with the widget.
+
+If you’re using a different framework or platform, Plaid offers [SDKs](https://plaid.com/docs/api/libraries/#link-client-sdks) tailored to various environments from which you can select the appropriate package for your application infrastructure.
+
 ## Integration Steps
 
 ### Step 1: Initiate Exchange Session with Plaid
 
 To begin, you will [create an exchange session](https://developers.dwolla.com/docs/balance/api-reference/open-banking/create-exchange-session-for-a-customer) for your Customer in Dwolla using the **Exchange Sessions API**. This session will specify Plaid as the **exchange partner**. The Exchange Partner ID for Plaid can be found by calling the [List Exchange Partners](https://developers.dwolla.com/docs/balance/api-reference/exchanges/list-exchange-partners) API endpoint.
 
+**About the `redirect-url` Field:**
+
+If have an **Android** and/or **iOS** app, you must include a platform-specific `redirect-url` in the request body when creating an exchange session. This URL determines where the user is redirected after completing the Plaid Link flow. The `redirect-url` is a **required** field only for Android and iOS apps. It is not needed for web-based implementations.
+
+- For **Android**, use the Android package name as the redirect-url value (e.g., `com.example.app123`).
+- For **iOS**, use a valid HTTP or HTTPS URL (e.g., `https://example.com/app123`) that can handle redirects in the app.
+
+Dwolla will validate the provided `redirect-url` based on these conventions and send the appropriate value to Plaid. If the redirect-url does not start with a valid protocol (https:// or http://), it will be assumed to be an Android package name.
+
 <AlertBar variation="info">
   Exchange Sessions are single-use. Once a user starts the IAV flow initiated by
   creation of a session, it becomes invalid and cannot be reused.
 </AlertBar>
 
-##### Example: Initiating an Exchange Session via Dwolla API using Dwolla Node SDK
+##### Example: Initiating an Exchange Session via Dwolla API (toggle for Node example)
```

Comment:
```suggestion
##### Example: Initiating an Exchange Session via Dwolla API
```
I'd remove the toggle for Node example since we'll likely want to support other languages in the future. The user could also self discover